### PR TITLE
feat: split security check out of lock functions

### PR DESCRIPTION
**Pull request by Julusian**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Fix

* **What is the current behavior?** (You can also link to an open issue here)

Some security warnings are thrown at startup in development. 

* **What is the new behavior (if this is a feature change)?**

Fixes these warnings. This is done mostly by splitting the security check out of the lock functions, with the result of the check being passed into the lock function instead

* **Other information**:

We have discussed removing a lot of the `ServerPlayoutAPI` as the `UserActionsAPI` should always be used.
I have done that as part of this, as otherwise each method needs updating to be hooked into the security correctly.

**Status**

- [ ] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [ ] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/server/api/lib.ts [hunk 1]
```diff
@@ -2,7 +2,24 @@ import { Meteor } from 'meteor/meteor'
 import { MethodContext } from '../../lib/api/methods'
 import { RundownPlaylistId, RundownPlaylist } from '../../lib/collections/RundownPlaylists'
 import { Rundown, RundownId } from '../../lib/collections/Rundowns'
-import { RundownPlaylistContentWriteAccess } from '../security/rundownPlaylist'
+import { RundownPlaylistContentAccess, RundownPlaylistContentWriteAccess } from '../security/rundownPlaylist'
+
+export interface VerifiedRundownPlaylistContentAccess extends RundownPlaylistContentAccess {
```
Comment (nytamin) [inline meteor/server/api/lib.ts:7]: I think this would benefit from some doc

File: meteor/server/api/playout/playout.ts [hunk 2]
```diff
@@ -985,7 +995,7 @@ export namespace ServerPlayoutAPI {
 		triggerWriteAccessBecauseNoCheckNecessary() // tmp
 
 		return runPlayoutOperationWithLock(
-			context,
+			null, // TODO: should security be done?
```
Comment (nytamin) [inline meteor/server/api/playout/playout.ts:998]: In the end? Yes, I think all methods should ideally be access controlled properly, even the callbacks from devices.
But I suppose that it's a bigger issue, let's cross that stream later.

**Review by nytamin — APPROVED** [review]
A few minor comments, Looks good!

File: meteor/server/api/playout/playout.ts (hunk 2 above)
Comment (Julusian) [reply meteor/server/api/playout/playout.ts:998]: This comment was already on one of these methods, and I don't think it work the same way as the other methods, so thought it best to do this for now.

**Pull request merged**